measurements/api/measurements.py

```diff
@@ -214,21 +250,42 @@ def input_cte(input_, domain, test_name):
             text('input.input LIKE :i').bindparams(i='%{}%'.format(input_)),
         )
 
-    if domain:
```

Comment:
Apart from the "if" this is just a reflow.

Reply:
The idea was that you could pass in either of the two or possibly both and the filter would apply to both.

By dropping the `if` and making it into an `else` this function is no longer equivalent to the previous function.

I suggest putting it back to how it was and applying the reflow to the original function.

Reply:
Ignore this last comment.